test/central_planner_test.py

```diff
@@ -898,6 +899,310 @@ def test_priority_no_decrease_with_multiple_updates(self):
         self.sch.add_task(worker=WORKER, task_id='D', priority=6)
         self.check_task_order(['A', 'B', 'D', 'C'])
 
+    def task_id_str(self, family, args):
+        return task_id_str(family, args)
+
+    def test_aggregation(self, aggregate_type='csv', expected_args='1,3'):
```

Comment:
Can you rename this like `_test_aggregation` and make one new test be `test_aggregation_csv` for sake of consistency?